cf/tasks/package-bits.sh

```diff
@@ -30,8 +30,9 @@ do
     mkdir tmp
 
     # kube configs
-    # helm charts
     unzip s3.kube-dist/scf-kube-*.zip -d tmp
+    # helm charts
+    unzip s3.helm-dist/scf-helm-*.zip -d tmp
```

Comment:
Please run this on your concourse and send me the resulting zip archive.
(On my local one it fails to upload to your minio, access denied)
(With a zip file in hand I should be able to complete the docs)